utils/zksync/era/token-library.ts

```diff
@@ -0,0 +1,19 @@
+import { getTokenCollection } from "@matterlabs/token-library";
+
+export const getTokensByNetworkId = async (networkId: number) => {
+  if (![270, 324, 280].includes(networkId)) throw new Error(`Network id ${networkId} is not supported`);
```

Comment:
This won't break new chains correct? They just don't have a token list for now

Reply:
Yeah, it's just for token library. This function just isn't used for other chains